homeassistant/components/bang_olufsen/const.py

```diff
@@ -10,27 +10,27 @@
 from homeassistant.components.media_player import MediaPlayerState, MediaType
 
 
-class BangOlufsenSource(StrEnum):
-    """Enum used for associating device source ids with friendly names. May not include all sources."""
-
-    URI_STREAMER = "Audio Streamer"
-    BLUETOOTH = "Bluetooth"
-    AIR_PLAY = "AirPlay"
-    CHROMECAST = "Chromecast built-in"
-    SPOTIFY = "Spotify Connect"
-    GENERATOR = "Tone Generator"
-    LINE_IN = "Line-In"
-    SPDIF = "Optical"
-    NET_RADIO = "B&O Radio"
-    LOCAL = "Local"
-    DLNA = "DLNA"
-    QPLAY = "QPlay"
-    WPL = "Wireless Powerlink"
-    PL = "Powerlink"
-    TV = "TV"
-    DEEZER = "Deezer"
-    BEOLINK = "Networklink"
-    TIDAL_CONNECT = "Tidal Connect"
+class BangOlufsenSource:
```

Comment:
This can still be an enum right?

Reply:
It can, but using the enum is a bit more confusing with having to use the "name" and "value" members

Reply:
Yeah, it's a bit messy. Honestly I think that the class should be removed altogether, most of its uses are at best mildly useful. I think I'll try to retain some of the functionality and remove it.

Reply:
Like, I think we can improve this more than we do now. If we can move parts to the lib, that's always great

Reply:
A future API update will remove some of the need for BangOlufsenSource, but I think the current state (of this PR) should be fine for now